### ISSUE TITLE
babelLoader.options goes missing

### ISSUE THREAD

**Issue by gertsonderby** [withspectrum/react-app-rewire-styled-components#3]
I have a simple setup - very nearly raw CRA react-app with styled-components, react-app-rewired and this lib. I set it up according to the readme. I am getting the following error output, however:
```
yarn start v0.27.5
$ react-app-rewired start
/home/gert/devel/familiarity/node_modules/react-app-rewire-styled-components/dist/index.js:9
  babelLoader.options.plugins = (babelLoader.options.plugins || []).concat([['styled-components', styledComponentsPluginOptions]]);
             ^

TypeError: Cannot read property 'options' of undefined
    at rewireStyledComponents (/home/gert/devel/familiarity/node_modules/react-app-rewire-styled-components/dist/index.js:9:14)
    at override (/home/gert/devel/familiarity/config-overrides.js:4:12)
    at Object.<anonymous> (/home/gert/devel/familiarity/node_modules/react-app-rewired/scripts/start.js:18:3)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
    at Function.Module.runMain (module.js:605:10)
    at startup (bootstrap_node.js:158:16)
    at bootstrap_node.js:575:3
error Command failed with exit code 1.
```
Running `yarn test` (which also uses react-app-rewired) runs without issues.

`config-overrides.js`:
```js
const rewireStyledComponents = require('react-app-rewire-styled-components');

module.exports = function override(config, env) {
  config = rewireStyledComponents(config, env);
  return config;
}
```
Versions:
    "react": "15.6.1",
    "react-dom": "15.6.1",
    "react-scripts": "1.0.11",
    "react-app-rewire-styled-components": "2.0.0",
    "react-app-rewired": "1.1.0",

**Comment by uberbryn** [comment]
I just ran into this today as well when deploying to now, but it runs fine locally. 🤔

**Comment by gertsonderby** [comment]
Mine has, so far, consistently worked on test, failed on start. I've not made a build yet, since I'm setting up the project still, very early days.

**Comment by amel-true** [comment]
With CRA 1.0.10 ("react-scripts": "1.0.10") works well

**Comment by DonnieWest** [comment]
Looks like react-scripts 1.0.11 broke this plugin with this commit https://github.com/facebookincubator/create-react-app/commit/a08eb3b44b47b3a1c9776e0859943f610ab50085

The babel loader is now in a slightly different place. Came across this while debugging another rewire and figured I'd drop this here

**Comment by mxstbr** [comment]
Yeah it makes sense that a CRA upgrade breaks this, let's see how @timarney fixes this upstream in any of his `react-app-rewire-x` plugins that adapt the Babel config?

**Comment by timarney** [comment]
I won't have time to look at this today but I added a utility method as a test a while back - `injectBabelPlugin` Idea is to to have that available for all plugins in one place for all.

It's here https://github.com/timarney/react-app-rewired/blob/master/packages/react-app-rewired/index.js

**Comment by geddski** [comment]
just ran into the same issue

**Comment by mxstbr** [comment]
Yeah let's use `injectBabelPlugin` do we don't have to keep this pkg in sync! PRs welcome

**Comment by timarney** [comment]
Just published react-app-rewired - 1.2.0 should fix the issue for 1.0.11 if you use injectBabelPlugin.

**Comment by mxstbr** [comment]
Thanks to @canuzunoglu we have the patch in and it's published as `1.0.2`, let me know if it doesn't work for you!
